python/ray/dashboard/modules/reporter/reporter_agent.py

```diff
@@ -436,6 +440,14 @@ async def CpuProfiling(self, request, context):
         )
         return reporter_pb2.CpuProfilingReply(output=output, success=success)
 
+    async def GpuProfiling(self, request, context):
```

Comment:
Is this automatically hooked up to the `GpuProfiling` proto rpc?
Don't see any grpc code for handling it.

Reply:
The proto is updated here: https://github.com/ray-project/ray/pull/53191/files#diff-1df7450350b7ffdc1923f9321c2df30c8217b6060106700e4920ba447eb7eb84

Reply:
yeah, i see the proto change, but im surprised that you don't have to "hook up" the class to that proto in some way

Reply:
oh yeah I think it just autogenerates the reporter stub with the new GpuProfile method